scripts/inference/cropland_mapping.py

```diff
@@ -41,29 +50,70 @@ def extract(self, image):
     miny = args.miny
     maxx = args.maxx
     maxy = args.maxy
+    epsg = args.epsg
 
     start_date = args.start_date
     end_date = args.end_date
 
-    spatial_extent = BoundingBoxExtent(minx, miny, maxx, maxy)
+    spatial_extent = BoundingBoxExtent(minx, miny, maxx, maxy, epsg)
     temporal_extent = TemporalContext(start_date, end_date)
 
-    backend = BackendContext(Backend.CDSE)
+    backend_context = BackendContext(Backend.FED)
+
+    connection = openeo.connect(
+        "https://openeo.creo.vito.be/openeo/"
+    ).authenticate_oidc()
 
     # Preparing the input cube for the inference
-    input_cube = worldcereal_preprocessed_inputs_gfmap(
-        connection=cdse_connection(),
-        backend_context=backend,
+    inputs = worldcereal_preprocessed_inputs_gfmap(
+        connection=connection,
+        backend_context=backend_context,
         spatial_extent=spatial_extent,
         temporal_extent=temporal_extent,
     )
 
-    # Start the job and download
-    job = input_cube.create_job(
-        title=f"Cropland inference BBOX: {minx} {miny} {maxx} {maxy}",
-        description="Cropland inference using WorldCereal, Presto and GFMAP classifiers",
-        out_format="NetCDF",
+    # Test feature computer
+    presto_parameters = {
+        "rescale_s1": False,  # Will be done in the Presto UDF itself!
```

Comment:
A little confusing, let's discuss this. If needed, we can also adapt the Presto UDF and remain consistent.